src/types.ts

```diff
@@ -1,32 +1,26 @@
 import type { Json, Keyring } from '@metamask/utils';
-import ObservableStore from 'obs-store';
 
 export type KeyringControllerArgs = {
-  keyringBuilders:
-    | { (): Keyring<Json>; type: string }
-    | ConcatArray<{ (): Keyring<Json>; type: string }>;
-
+  keyringBuilders: { (): Keyring<Json>; type: string }[];
   cacheEncryptionKey: boolean;
```

Comment:
Should we mark this as optional as well? Since it was effectively optional in older versions